<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_fondo">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QWidget" name="widget_contenedor" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <widget class="QFrame" name="frame_planos">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>281</width>
              <height>761</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(74, 111, 165);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="plano_Axial">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="plano_Sagital">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="plano_Coronal">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="widget_spinBoxes" native="true">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>0</y>
              <width>51</width>
              <height>761</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(225, 225, 225);
background-color: rgb(232, 240, 254);</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QSpinBox" name="spinBox_Axial">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="Slider_Axial">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_Sagital">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="Slider_Sagital">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_Coronal">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="Slider_Coronal">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QFrame" name="frame_Derecho">
            <property name="geometry">
             <rect>
              <x>360</x>
              <y>0</y>
              <width>451</width>
              <height>721</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(180, 200, 240);
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="boton_SubirArchivo">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>30</y>
               <width>201</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(17, 170, 255);
;
color: rgb(255, 255, 255);

 border-radius: 10px; 
</string>
             </property>
             <property name="text">
              <string>Subir Archivo</string>
             </property>
            </widget>
            <widget class="QLabel" name="Info_Estudio">
             <property name="geometry">
              <rect>
               <x>14</x>
               <y>85</y>
               <width>421</width>
               <height>421</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(238, 238, 238);</string>
             </property>
             <property name="text">
              <string>   Informaci√≥n del Estudio (metadatos ordenados) 
 </string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>550</y>
               <width>407</width>
               <height>171</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="2">
               <widget class="QPushButton" name="boton_ModificarMetadatos">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(17, 170, 255);
;
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Modificar Metadatos</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="boton_GuardarEstudio">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(170, 17, 255);
;
color: rgb(255, 255, 127);</string>
                </property>
                <property name="text">
                 <string>Guardar Estudio</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="boton_transf_dcm_nii">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(17, 170, 255);
;
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Transformar DICOM a NIfTI</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="boton_Limpiar">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(85, 170, 255);</string>
                </property>
                <property name="text">
                 <string>Limpiarüßπ</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="boton_volver">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 127);</string>
                </property>
                <property name="text">
                 <string>VOLVER</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
